archivers/star/Portfile

```diff
@@ -1,51 +1,105 @@
+                    \n \
+                    \nPLEASE NOTE:  Unlike the pax archiver provided by macOS, \
+                    Schily's pax (or spax) archiver is fully compliant with the POSIX \
+                    1003.1-2001 standard!  In other words, unlike the pax \
+                    archiver provided by macOS, the Schily pax (or spax) archiver will \
+                    actually create and read archives in the POSIX 1003.1-2001 pax \
+                    archive format! \
+                    \n \
+                    \nAlso included is the Schily tartest utility used to test whether a \
+                    tar archive is POSIX compliant and the star_sym utility which \
+                    reads a inode database file from a previous incremental restore via \
+                    the star archiver and prints the database in a human readable format. \
+                    \n \
+                    \nOmitted from the port are the mt, smt and rmt binaries to control \
+                    magnetic tape drives.
+
+homepage            https://cdrtools.sourceforge.net/private/star.html
+master_sites        sourceforge:s-tar
+checksums           sha1    2ee419a1167a54701183b79e75ce3e11de9872a5 \
+                    rmd160  43ef9e17244cbb63da91a1cb56a57d3d8519da30 \
+                    sha256  1b18662ebae6c3509020953d4464e654c7ded4752774de987d8dc307dc43f776 \
+                    size    1154422
 
-depends_build       port:smake
+patchfiles          patch-RULES__rules.prg.diff \
+                    patch-gnutar-manpage.diff
+
+use_bzip2           yes
 
 worksrcdir          star-${version}
 
-use_configure       no
+#
+# Note:  smake no longer builds the star port properly. Must use a modern version of gmake
+#
+depends_build-append     port:gmake
 
-# smake does not support the -j flag
-build.jobs          -1
+build.cmd           ${prefix}/bin/gmake
 
-build.cmd           ${prefix}/bin/smake
+#
+# Must specify INS_BASE and INS_RBASE in both the build and destroot phases
+#
+build.args          GMAKE_NOWARN=true \
+                    INS_BASE=${destroot}${prefix} \
+                    INS_RBASE=${destroot}${prefix}
+                    
+universal_variant   no
 
 destroot.destdir    INS_BASE=${destroot}${prefix} \
+                    INS_RBASE=${destroot}${prefix} \
                     MANDIR=man
+
 post-destroot {
     system "cd ${destroot}${prefix}/bin/ && \
-        rm -f gnutar mt tar && \
-        mv tartest startest"
+        rm -f mt smt rmt tar && \
+        mv gnutar sgnutar"
+        
     system "cd ${destroot}${prefix}/share/man && \
-        rm -rf man1/gnutar.1 man1/match.1 man1/rmt.1 man3 \
+        rm -f man1/mt.1 man1/smt.1 man1/rmt.1 man1/match.1 \
         man5/makerules.5 man5/makefiles.5 && \
-        mv man1/tartest.1 man1/startest.1"
-    file delete -force ${destroot}${prefix}/lib ${destroot}${prefix}/include
+        rm -rf man3 && \
+        mv man1/gnutar.1 man1/sgnutar.1"
```

Comment:
```suggestion
    foreach f {mt smt rmt tar} {
        file delete -force ${destroot}${prefix}/bin/$f
    }

    file rename \
        ${destroot}${prefix}/bin/gnutar \
        ${destroot}${prefix}/bin/sgnutar

    foreach f {man1/mt.1 man1/smt.1 man1/rmt.1 man1/match.1 man5/makerules.5 man5/makefiles.5 man3} {
        file delete -force ${destroot}${prefix}/share/man/$f
    }

    file rename \
        ${destroot}${prefix}/share/man/man1/gnutar.1 \
        ${destroot}${prefix}/share/man/man1/sgnutar.1
```

Reply:
Thanks!  Your suggested code is easier to read.  I will try it.  I cannot commit these changes here.  When I try I see "Applying suggestions on deleted lines is currently not supported.".  Bummer.

Reply:
@RobK88 you still may change it locally and do a force push to your branch ;)

Reply:
Applied and pushed all the suggested changes.